tests/integration_tests/test_login.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+
+from .conftest import (
+    connect_and_sign_in, connect_client, perform_login, read_until_command
+)
+
+pytestmark = pytest.mark.asyncio
```

Comment:
Is this used anywhere?

Reply:
Yes, see the `Markers` section: https://pypi.org/project/pytest-asyncio